app/views/devise/registrations/new.html.haml

```diff
@@ -41,6 +41,6 @@
     = f.input_field :password_confirmation, class: 'form-control'
 
   .form-group
-    = f.button :submit, 'Sign up', class: 'button'
+    = govuk_button() { 'Sign up' }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)